playbooks/templates/rax-maas/ironic_compute_check.yaml.j2

```diff
@@ -0,0 +1,20 @@
+{% set label = "ironic_compute_check" %}
+{% set check_name = label+'--'+inventory_hostname %}
+type        : agent.plugin
+label       : "{{ check_name }}"
+period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled    : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : ["{{ maas_plugin_dir }}/nova_service_check.py", "--host", "{{ ansible_nodename }}", "{{ internal_vip_address }}"]
+alarms      :
+    ironic_compute_status :
+        label                   : ironic_compute_status--{{ inventory_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan_override[label] | default(maas_notification_plan) }}"
+        disabled                : {{ (('nova_compute_status--'+inventory_hostname) | match(maas_excluded_alarms_regex)) | ternary('true', 'false') }}
```

Comment:
The disabled name should also match the alarm name, `ironic_compute_status`, instead of nova_compute_status.